plugins/dynamix/include/update.parity.php

```diff
@@ -30,6 +30,7 @@ function month($m) {
     $month = $_POST['month'] ?? '*';
     $day   = $_POST['day'] ?? '*';
     $write = $_POST['write'] ?? '';
+	$write = ($write == "CORRECT") ? "" : "NOCORRECT";
```

Comment:
tabs vs spaces, causes strange indenting

Reply:
Fixed.  I normally tab in all my work.  I'm not used to spaces.